src/Workspaces/Core/MSBuild/MSBuild/MSBuildWorkspace.cs

```diff
@@ -137,6 +137,15 @@ public bool SkipUnrecognizedProjects
             set => _loader.SkipUnrecognizedProjects = value;
         }
 
+        /// <summary>
+        /// 
```

Comment:
"Whether to generate a binary log in the current working directory when loaded" ... I am not good with words